README.md

```diff
@@ -1,3 +1,33 @@
 # moj-module-action-group
 
-A module that lets you create Action Groups in Azure
\ No newline at end of file
+A module that lets you create Action Groups in Azure
+
+## Variables
+
+### Contextual variables
+
+#### `location`
+The azure data center location.
+
+#### `env`
+The application insights environment. Use `prod` for production, `sandbox` for sandbox, etc.
+
+#### `subscription`
+The application insights subscription name, such as `cmc-prod`.
```

Comment:
isn't this not used? cmc-prod isn't a subscription its an app insights resource

Reply:
I'll test re: not being used; but definitely the wrong example